namesilo.php

```diff
@@ -781,7 +877,7 @@ public function manageAddRow(array &$vars)
             $this->view->setDefaultView(self::$defaultModuleView);
 
             // Load the helpers required for this view
-            Loader::loadHelpers($this, [ 'Form', 'Html', 'Widget' ]);
+            Loader::loadHelpers($this, ['Form', 'Html', 'Widget']);
```

Comment:
Moving the helpers to the top causes an  "Undefined property" error, the helpers must be loaded after loading the view, otherwise it seems that the helpers are removed when the view is loaded.